### PR TITLE
Feature/update databricks getting started

### PR DESCRIPTION
**Pull request by amychen1776**
## Description & motivation
Databricks just realized a new functionality that allows folks to upload new data without going into the Data Science section. This creates a much more seamless experience for students going through the getting started guide so I've updated it.

### CODE REVIEW

File: website/docs/tutorial/getting-set-up/setting-up-databricks.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Before starting this tutorial, you will need the following:
     <Lightbox src="/img/databricks_tutorial/images/quick_create_stack.png" title="Create AWS resources" />
     </div>
 
-    *Tip:* If you get a session error and don’t get redirected to this page, do not worry, go back to the Databricks UI and create a workspace from the interface. All you have to do is click **create workspaces**, choose the quickstart, fill out the form and click **Start Quickstart**`.
+    *Tip:* If you get a session error and don’t get redirected to this page, do not worry, go back to the Databricks UI and create a workspace from the interface. All you have to do is click **create workspaces**, choose the quickstart, fill out the form and click **Start Quickstart**.
```
Comment (runleonarun) [inline website/docs/tutorial/getting-set-up/setting-up-databricks.md]: Let's use the standard tip format. It breaks the page up nicely.
```suggestion
:::tip
If you get a session error and don’t get redirected to this page, do not worry, go back to the Databricks UI and create a workspace from the interface. All you have to do is click **create workspaces**, choose the quickstart, fill out the form and click **Start Quickstart**.
:::
```

File: website/docs/tutorial/getting-set-up/setting-up-databricks.md [hunk 2]
```diff
@@ -85,69 +85,63 @@ Congrats! You are now ready to start working in the workspace.
 <Lightbox src="/img/databricks_tutorial/images/welcome_page.png" title="Welcome to the Databricks Workspace" />
 </div>
 
-Next up, you will need a cluster for our training. A spark cluster is going to be your compution resource for running your analytics workloads. We will be using it to run our queries.
```
Comment (runleonarun) [inline website/docs/tutorial/getting-set-up/setting-up-databricks.md:88]: This is amazing you were able to remove all of these steps!!! 🤩

File: website/docs/tutorial/getting-set-up/setting-up-databricks.md [hunk 3]
```diff
@@ -85,69 +85,63 @@ Congrats! You are now ready to start working in the workspace.
...
-    </div>
-4. Now let's upload our first csv. Drag and drop the `jaffle_shop_customers.csv` file into the UI. 
+5. Once the SQL Endpoint is up, click on the `Create` and then `Table` on the drop down menu. 
     <div style={{maxWidth: '400px'}}>
-    <Lightbox src="/img/databricks_tutorial/images/create_new_table.png" title="Upload csv file" />
+    <Lightbox src="/img/databricks_tutorial/images/create_table_using_databricks_SQL.png" title="Create Table Using Databricks SQL" />
     </div>
-5. Click **Create Table with UI**, then select your cluster from the dropdown of available clusters. Click **Preview Table**.
+
+6. Let's load the Jaffle Shop Customers data first. Drop in the jaffle_shop_customers.csv file into the UI.
```
Comment (runleonarun) [inline website/docs/tutorial/getting-set-up/setting-up-databricks.md]: Let's make filenames `code`.
```suggestion
6. Let's load the Jaffle Shop Customers data first. Drop in the `jaffle_shop_customers.csv` file into the UI.
```

File: website/docs/tutorial/getting-set-up/setting-up-databricks.md [hunk 4]
```diff
@@ -85,69 +85,63 @@ Congrats! You are now ready to start working in the workspace.
...
+    </div>
+2. We will be setting up a SQL endpoint now.  Select `SQL Endpoints` from the left hand side console.  You will see that a default SQL Endpoint exists.  
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/sql_endpoints.png" title="SQL Endpoints" />
+    </div>
+3. Click **Start** on the Starter Endpoint.  This will take a few minutes to get the necessary resources spun up.
 
-1. Download the three CSV files locally that you will need for this tutorial. You can find them here:
+4. While you're waiting, download the three CSV files locally that you will need for this tutorial. You can find them here:
     - [jaffle_shop_customers.csv](http://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_customers.csv)
```
Comment (runleonarun) [inline website/docs/tutorial/getting-set-up/setting-up-databricks.md]: I'm having trouble downloading from this link. Nothing happens. But when I copy and past the URL into a new tab, I get the download. I'm using Chrome.  Should it be `https`?

File: website/static/img/databricks_tutorial/images/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Welcome to your new dbt project!
+
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/README.md]: @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/static/img/databricks_tutorial/images/analysis/total_revenue.sql [hunk 6]
```diff
@@ -0,0 +1,12 @@
+with payments as (
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/analysis/total_revenue.sql]: Same comment as above:
> @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/static/img/databricks_tutorial/images/data/employees.csv [hunk 7]
```diff
@@ -0,0 +1,5 @@
+employee_id, email, customer_id
+3425, mike@jaffleshop.com, 1
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/data/employees.csv]: Same comment as above:
> @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/static/img/databricks_tutorial/images/dbt_project.yml [hunk 8]
```diff
@@ -0,0 +1,39 @@
+
+# Name your project! Project names should contain only lowercase characters
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/dbt_project.yml]: Same comment as above:
> @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/static/img/databricks_tutorial/images/models/all_dates.sql [hunk 9]
```diff
@@ -0,0 +1,10 @@
+{{ config (
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/models/all_dates.sql]: Same comment as above:
> @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/static/img/databricks_tutorial/images/macros/cents_to_dollars.sql [hunk 10]
```diff
@@ -0,0 +1,3 @@
+{% macro cents_to_dollars(column_name, decimal_places=2) -%}
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/macros/cents_to_dollars.sql]: Same comment as above:
> @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/static/img/databricks_tutorial/images/models/marts/core/dim_customers.sql [hunk 11]
```diff
@@ -0,0 +1,29 @@
+with customers as (
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/models/marts/core/dim_customers.sql]: Same comment as above:
> @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/static/img/databricks_tutorial/images/models/marts/core/fct_orders.sql [hunk 12]
```diff
@@ -0,0 +1,30 @@
+with orders as  (
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/models/marts/core/fct_orders.sql]: Same comment as above:
> @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/static/img/databricks_tutorial/images/models/staging/jaffle_shop/jaffle_shop.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
+{% docs order_status %}
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/models/staging/jaffle_shop/jaffle_shop.md]: Same comment as above:
> @amychen1776 is this starter content? Maybe we can create a databricks folder in the tutorial directory? It feels very out-of-place in the `static/img` directory.

File: website/snippets/tutorial-initiate-project.md [hunk 14]
```diff
@@ -27,7 +27,7 @@ select * from jaffle_shop_customers
 <div>
 
 ```sql
-select * from jaffle_shop.customers
+select * from default.jaffle_shop_customers
```
Comment (runleonarun) [inline website/snippets/tutorial-initiate-project.md]: I am not seeing this showing up in the [output on the staging server](https://deploy-preview-1461--docs-getdbt-com.netlify.app/tutorial/getting-set-up/setting-up-databricks#initialize-your-dbt-project). 🤔

**Review by runleonarun — COMMENTED** [review]
Hey Amy! This looks mostly there. I'm really loving how you've removed so many steps!

A few feedback comments:
* Can the content that is not images be removed form `website/static/img/databricks_tutorial/` and put somewhere else like in the tutorial directory? I'm also not sure what this content is being used for? I don't see any links from the tutorial to it. 
* See other minor suggestions inline.

File: website/docs/tutorial/getting-set-up/setting-up-databricks.md (hunk 2 above)
Comment (amychen1776) [reply website/docs/tutorial/getting-set-up/setting-up-databricks.md:88]: I know! Also we don't have to change interfaces. It's so amazing

File: website/static/img/databricks_tutorial/images/README.md (hunk 5 above)
Comment (amychen1776) [reply website/static/img/databricks_tutorial/images/README.md]: Sorry! This is my bad; I didn't realize I accidentally copied over the folders and files from another window 😢 Fixing that now! (serves me right for trying to update dbt init at the same time)

File: website/snippets/tutorial-initiate-project.md (hunk 14 above)
Comment (amychen1776) [reply website/snippets/tutorial-initiate-project.md]: @runleonarun Could you check the snippet for me? I can't seem to figure out why it's not loading :(

Comment (amychen1776) [reply website/snippets/tutorial-initiate-project.md]: Oh wait, it's showing up locally for me 
image

File: website/docs/tutorial/getting-set-up/setting-up-databricks.md (hunk 4 above)
Comment (amychen1776) [reply website/docs/tutorial/getting-set-up/setting-up-databricks.md]: Yup! You're correct; I just fixed it.

File: website/snippets/tutorial-initiate-project.md (hunk 14 above)
Comment (runleonarun) [reply website/snippets/tutorial-initiate-project.md]: Oh @amychen1776  it's under Redshift! Is that intentional?

File: website/static/img/databricks_tutorial/images/packages.yml [hunk 15]
```diff
@@ -0,0 +1,3 @@
+packages:
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/packages.yml]: @amychen1776 Is this packages.yml also OK to remove?

File: website/static/img/databricks_tutorial/images/dbt_project.yml [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: 'jaffle_shop'
+version: '1.0.0'
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/dbt_project.yml]: @amychen1776 You may have moved this locally, but I still see it. Is this also OK to remove?

File: website/static/img/databricks_tutorial/images/.gitignore [hunk 17]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (runleonarun) [inline website/static/img/databricks_tutorial/images/.gitignore]: @amychen1776 This one can also be removed!

Comment (amychen1776) [reply website/static/img/databricks_tutorial/images/.gitignore]: Got it!

File: website/static/img/databricks_tutorial/images/dbt_project.yml (hunk 16 above)
Comment (amychen1776) [reply website/static/img/databricks_tutorial/images/dbt_project.yml]: Missed this one!

File: website/static/img/databricks_tutorial/images/packages.yml (hunk 15 above)
Comment (amychen1776) [reply website/static/img/databricks_tutorial/images/packages.yml]: removed!

**Review by runleonarun — APPROVED** [review]
🚢 it!

**Pull request merged**